service/src/main/resources/changelog/changesets/20240126_github_method_details_batch_or_tag.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: "1"
+      author: kpierre
+      changes:
+        - createTable:
+            tableName: github_method_details
+            columns:
+              - column:
+                  name: repository
+                  type: varchar(100)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: organization
+                  type: varchar(100)
+                  constraints:
+                    nullable: true
+              - column:
+                  name: path
+                  type: varchar(1000)
```

Comment:
TOL: Should this and above `varchar` also be `text`? Is there a benefit of `varchar` over `text`?